torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -71,7 +71,7 @@ C10_REGISTER_TYPED_CLASS(TimerRegistry, c10::kCPU, CpuTimer);
 } // namespace
 
 Reducer::Reducer(
-    std::vector<std::vector<at::Tensor>> replicas,
+    std::vector<at::Tensor> replicas,
```

Comment:
The var name `replicas` was referring to replicas of model parameters. Since now there is only one replica, I think we can rename this var to `params`, where which tensor is a parameter.

Reply:
Done